packages/contracts/contracts/optimistic-ethereum/ovm/FullStateManager.sol

```diff
@@ -116,37 +152,45 @@ contract FullStateManager is StateManager {
      ******************/
 
     /**
-     * @notice Attaches some code contract to the desired OVM contract. This allows the Execution Manager
-     *         to later on get the code contract address to perform calls for this OVM contract.
+     * Attaches some code contract to the desired OVM contract. This allows the Execution Manager
+     * to later on get the code contract address to perform calls for this OVM contract.
      * @param _ovmContractAddress The address of the OVM contract we'd like to associate with some code.
      * @param _codeContractAddress The address of the code contract that's been deployed.
      */
     function associateCodeContract(
         address _ovmContractAddress,
         address _codeContractAddress
-    ) public {
+    )
+        public
+    {
         ovmAddressToCodeContractAddress[_ovmContractAddress] = _codeContractAddress;
         codeContractAddressToOvmAddress[_codeContractAddress] = _ovmContractAddress;
     }
 
     /**
-     * @notice Marks a contract as newly created. Unused within this implementation.
+     * Marks a contract as newly created. Unused within this implementation.
      * @param _ovmContractAddress Address to mark as newly created.
      */
     function associateCreatedContract(
```

Comment:
There's a possibility that I commented on this in a different PR or messaged you privately or something, but in case I didn't -- is there a better word for this functionality? Of course this is not used in the `FullStateManager`, but still think the word `associate` doesn't do the functionality justice

Reply:
Changed to `registerCreatedContract`. Seems good enough for now.